src/test/java/com/wikia/webdriver/common/dataprovider/ads/AdsDataProvider.java

```diff
@@ -97,61 +97,63 @@ private AdsDataProvider() {
   public static Object[][] skinWithTheme() {
     return new Object[][]{
         {
-            WIKI_REGULAR, "Skin",
-            new Dimension(1200, 1000),
-            NO_SKIN_LEFT, NO_SKIN_RIGHT,
-            null,
-            null
-        }, {
+//            WIKI_REGULAR, "Skin",
+//            new Dimension(1200, 1000),
+//            NO_SKIN_LEFT, NO_SKIN_RIGHT,
+//            null,
+//            null
+//        }, {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3ACommentedOutCodeLine)